CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.8.2 
+- Realtime Environment always has a clock to prevent agents that define callbacks only from terminating
+- Environment time in real time is now based on system time, decoupling it from the simpy process
```

Comment:
Possibly link other related issues such as #19

Reply:
#19 is already merged, but I linked #14 in the description